docs/pages/desktop-access/introduction.mdx

```diff
@@ -4,59 +4,70 @@ description: Demonstrates how you can manage access to Windows desktops with Tel
+Windows Desktop Service typically makes an outbound connection to the Teleport
+Proxy Service and maintains a reverse tunnel that is used to power remote
+desktop sessions. Since the only networking requirement is a single outbound
+connection to the Teleport Proxy, you are free to deploy as many desktop
+services in as many different network segments as you need - Teleport will take
+care of selecting a route to your Windows host at connection time.
 
 ![Manage access to Windows through Teleport](../../img/desktop-access/windows-desktop-service-overview.png)
 
-It's worth noting that the Teleport Windows Desktop Service implements a minimal set 
-of remote desktop protocol features to minimize security vulnerabilities for Windows
-computers. Because secure access is the top priority, the Windows Desktop Service 
-might not be as performant as other RDP clients. 
+You can configure Teleport Windows Desktop Service to control access for the following
+scenarios:
 
-You should use the Teleport Windows Desktop Service to manage access 
-to Windows computers where you store or manipulate your most sensitive information, 
-rather than as a direct replacement for tools that provide general purpose access 
-to Windows computers.
+- Domain users who access computers that are joined to an Active Directory domain.
+- Local users who access computers that aren't joined to an Active Directory domain.
 
-## Getting started
+In both cases, Teleport authenticates by presenting a short lived certificate to
+the RDP server in the form of a virtual smart card - no passwords are involved.
 
-You can configure Teleport Windows Desktop Service to control access for the following 
-scenarios:
+Each approach offers its own pros and cons.
 
-- Local users who access computers that aren't joined to an Active Directory domain.
-- Domain users who access computers that are joined to an Active Directory domain.
+Teleport's support for domain users works with standard Active Directory smart
+card support, which comes with several benefits:
+
+- automatic inventory management via an LDAP integration
+- congfiguration can be managed centrally via group policy
+- no Teleport software runs on the Windows hosts
+
+The downside of the Active Directory setup is that Windows PKI can be difficult
+to configure and troubleshoot, and some environments require large configuration
+changes in order to support smart card authentication.
+
+Teleport's support for local users requires the installation of a separate
+package on each host that you wish to provide access to. System configuration is
+performed by the installer rather than via group policy. This approach is easier
+to set up and easier to troubleshoot, and also provides unique features like the
+ability to create users on-demand. The lack of automatic LDAP-based discovery
+and the requirement to install software on each host makes this approach less
+attractive for large environments.
```

Comment:
Do we want to mention somewhere that you can automate it using installer in CLI with something like Terraform?